src/js/validator.js

```diff
@@ -0,0 +1,50 @@
+const Validate = {
+    validResult: { validated: true, warnings: [] },
+
+    validateForm(form) {
+        this.validResult = { validated: true, warnings: [] };
+        if (form.title.length < 1 || form.title.length > 250)
+            this.updateResult('tutuł nie może mieć mniej niż 1 znaku oraz więcej niż 250');
+        if (form.questions.length < 1)
+            this.updateResult('tutuł nie może mieć mniej niż 1 znaku oraz więcej niż 250');
```

Comment:
Tu to samo, a w tym drugim coś się nie zgadza.

Reply:
done